File: RxTools-library/src/main/java/com/vondear/rxtools/view/popupwindows/tools/RxPopupViewTool.java
package com.vondear.rxtools.view.popupwindows.tools;

import java.util.Locale;

class RxPopupViewTool {

    static boolean isRtl() {
        Locale defLocal = Locale.getDefault();
        return Character.getDirectionality(defLocal.getDisplayName(defLocal).charAt(0))
                == Character.DIRECTIONALITY_RIGHT_TO_LEFT;
    }
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/popupwindows/tools/RxPopupViewTool.kt
package com.tamsiree.rxui.view.popupwindows.tools

import java.util.*

/**
 * @author Tamsiree
 */
internal object RxPopupViewTool {
    val isRtl: Boolean
        get() {
            val defLocal = Locale.getDefault()
            return (Character.getDirectionality(defLocal.getDisplayName(defLocal)[0])
                    == Character.DIRECTIONALITY_RIGHT_TO_LEFT)
        }
}